.github/workflows/continuous-build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         os:
-          - macos-latest
+          - macos-13
```

Comment:
Any reason to remove `macos-latest`? Why is it changed to `macos-13`, does having both help with testing all cases?

Reply:
this was changed @wangzlei here to block the pipelines https://github.com/aws/aws-xray-sdk-ruby/commit/26ae7b08c1799a8fd44718f658390d7ab3b5425b